Vue/vaahtwo/pages/public/setup/intall/Configuration.vue

```diff
@@ -93,11 +93,13 @@ onMounted(async () => {
                       <h5 class="text-left p-1 title is-6">App/Website Name</h5>
                       <div class="p-input">
                           <InputText v-model="store.config.env.app_name"
-                                     placeholder="App/Website Name"
+                                     placeholder="VaahCMS"
```

Comment:
@we-vikram-wri231 the place should remain: `Enter your website name or app name`, and a helper message and validation for `Only Alpha Numeric` value are allowed